### DISCUSSION TITLE
Clone SQL Database or multiple tables to MariaDB?

### DISCUSSION THREAD

**Discussion by KevinBurton** [dbeaver/dbeaver#11503] [Q&A]
**Is your feature request related to a problem? I see from DBeaver that there is an option to export data from individual tables from SQL Server. Then I could import that data into a MariaDB database/table. I basically want to be able to clone the "schema"  for a table including the table name AND copy the data in that table from MS SQL to MariaDB. Ideally I would like to do this for all the tables in the database/schema. I would like to be able to copy and paste a database from SQL Server to MariaDB. This is doesn't seem possible right now, as I just get an error message when I try it.

**Comment by kseniiaguzeeva** [comment]
It is a big dream of many users, but cannot be done now. Please use data transfer option to simplify data migration https://github.com/dbeaver/dbeaver/wiki/Data-migration.
